### PR TITLE
docs:  add  examples for each command

### PR DESCRIPTION
**Pull request by afmika**
#### Motivation and context

Getting started with `meta` cli should be easy

#### Migration notes

None

### Checklist

- [ ] The change come with new or modified tests
- [ ] Hard-to-understand functions have explanatory comments
- [ ] End-user documentation is updated to reflect the change

### CODE REVIEW

**Review by Yohe-Am — DISMISSED** [review]

File: examples/typegraphs/http-runtime.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# skip:start
```
Comment (destifo) [inline examples/typegraphs/http-runtime.py:1]: Do we need to skip the imports? shouldn't we keep them as a reference incase the user imports from the wrong file/lib?

**Review by destifo — APPROVED** [review]

File: examples/typegraphs/http-runtime.py (hunk 1 above)
Comment (zifeo) [reply examples/typegraphs/http-runtime.py:1]: This was a not optimal solutions, ideally, we should have the import small so the core the typegraph is the focus point, yet having a fully complete code

File: website/docs/reference/runtimes/temporal/index.mdx [hunk 2]
```diff
@@ -1,13 +1,82 @@
...
+An interesting use case is to dynamically describe the operations you want to expose, this enables reusing typegraphs accross different projects or even building a small framework around it.
+
+Here is a simple example of a typegraph that takes some value from an environment variable.
+
+<SDKTabs>
+    <TabItem value="python">
+```python
+from typegraph import t, typegraph, Policy, Graph
+from typegraph.providers.temporal import TemporalRuntime
+import os
```
Comment (zifeo) [inline website/docs/reference/runtimes/temporal/index.mdx:28]: Can we externalize any valid source code so everything is testing and snapshotted?

**Review by zifeo — APPROVED** [review]

File: website/docs/reference/runtimes/temporal/index.mdx (hunk 2 above)
Comment (afmika) [reply website/docs/reference/runtimes/temporal/index.mdx:28]: Will fix that in #676

**Pull request merged**